src/test/java/seedu/address/testutil/PersonBuilder.java

```diff
@@ -23,7 +23,7 @@ public class PersonBuilder {
     public static final String DEFAULT_PHONE = "85355255";
     public static final String DEFAULT_EMAIL = "amy@gmail.com";
     public static final String DEFAULT_ADDRESS = "123, Jurong West Ave 6, #08-111";
-    public static final String DEFAULT_FEES = "564";
+    public static final String DEFAULT_FEES = "100";
```

Comment:
Why does this need to be changed? Can't see reason

Reply:
Fixed